Makefile

```diff
@@ -0,0 +1,47 @@
+BOLD := \033[1m
+RESET := \033[0m
+
+# -- Docker
+COMPOSE              = docker-compose
+COMPOSE_RUN          = $(COMPOSE) run --rm
+COMPOSE_RUN_APP      = $(COMPOSE_RUN) app
+
+# ==============================================================================
+# RULES
+
+default: help
+
+bootstrap: ## Bootstrap nextcloud project
+bootstrap: \
+	build \
+	install
+.PHONY: bootstrap
+
+build: ## Build the app container
+	@$(COMPOSE) build nextcloud;
+.PHONY: build
+
+install: ## Install Nextcloud
+	@$(COMPOSE) up -d db
+	@echo "Wait for postgresql to be up..."
+	@$(COMPOSE_RUN) dockerize -wait tcp://db:5432 -timeout 60s
+	@$(COMPOSE_RUN) nextcloud-install;
+.phony: install
+
+run: ## start the development server using Docker
+	@$(COMPOSE) up -d nextcloud
```

Comment:
You can start the app even if the database is not ready to accept connections?

Reply:
good question, apparently yes.

Reply:
I start the db first now